src/router.ts

```diff
@@ -94,4 +101,51 @@ router.get("/raw-data/:id", async (req: Request, res: Response) => {
 	);
 });
 
+router.get(

```

Comment:
Is there a particular reason why `raw-data` endpoints are dumped here? There's `RawDataController.ts` in upstream where all endpoints should be put there, and `router.ts` is just for importing controllers.

Reply:
Has been merged and moved to RawDataController